README.md

```diff
@@ -113,6 +113,7 @@ const createComponent = createComponentFactory({
   declarations: [],
   entryComponents: [],
   componentProviders: [], // Override the component's providers
+  overrideModules: [], // Override modules [[SomeModule, {set: {...}}], [SomeOtherModule, {remove: {...}}]]
```

Comment:
Please update the future docs in the `docs` folder.

Reply:
Hi, 

I updated testing-components.md is it enough?

Reply:
Yes, thanks.